neptune/projects.py

```diff
@@ -412,14 +414,25 @@ def create_experiment(self,
             if os.path.isfile(main_file):
                 entrypoint = normalize_file_name(os.path.basename(main_file))
                 upload_source_entries = [
-                    UploadEntry(os.path.abspath(main_file), normalize_file_name(os.path.basename(main_file)))
+                    UploadEntry(abspath(main_file), normalize_file_name(os.path.basename(main_file)))
```

Comment:
You imported abspath, but not basename, I think it's confusing and it's better to leave os.path.abspath for reading clarity